plugin/storage/s3/s3_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"os"
```

Comment:
Hope we don't actually run these tests as part of GitHub action.

Reply:
It's skipped by `t.Skip()`. Added comments.